input/pagecontent/reference.md

```diff
@@ -330,18 +410,18 @@ Although it appears the quantity itself is being set to a coded value, this expr
   * valueQuantity from http://hl7.org/fhir/ValueSet/distance-units
   ```
 
-### Paths
+### FSH Paths
 
-FSH path grammar allows you to refer to any element of a profile, extension, or instance, regardless of nesting. Paths also provide a grammar for addressing elements of a StructureDefinition (SD) directly. Here are a few examples of how paths are used in FSH:
+FSH path grammar allows you to refer to any element of a profile, extension, or instance, regardless of nesting. Here are examples of things paths can refer to:
 
-* To refer to a top-level element such as the 'code' element in Observation
-* To refer to a nested element, such as the 'method.text' element in Observation
-* To address a particular item in a list or array
-* To refer to individual elements inside choice elements (e.g., onsetAge in onset[x])
-* To pick out an individual item within a multiple choice reference, such as Observation in Reference(Observation or Condition)
-* To refer to an individual slice within a sliced array, such as the SystolicBP component within a blood pressure
-* To set metadata elements in an SD, like 'active' and 'experimental'
-* To address properties of ElementDefinitions nested within an SD, such as 'maxLength' property of string-type elements
+* Top-level elements such as the `code` element in Observation
+* Nested elements, such as the `method.text` element in Observation
+* Elements in a list or array by index
+* Individual data types of choice elements, such as `onsetAge` in onset[x]
+* Individual items within a multiple choice reference, such as Observation in `Reference(Observation or Condition)`
+* Individual slices within a sliced array, such as the `SystolicBP` component within blood pressure
+* Metadata elements in an SD, like `active` and `experimental`
+* Properties of ElementDefinitions nested within an SD, such as `maxLength` property of string elements
 
 In the following, the various types of path references are discussed.
 
```

Comment:
That's a good idea, and it provides a place to talk about omitting the name of the resource.

#### Top-Level Paths

The path to a top-level element is denoted by the element's name. Because paths are used within the context of a FSH definition or instance, the path does not include the known context. For example, when defining a profile on Observation, the path to Observation.code is denoted as `code`.